Packs/FeedCyjax/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Cyjax Feed",
     "description": "This pack is used to pull indicators of compromise from the Cyjax Threat Intelligence Platform.",
     "support": "partner",
```

Comment:
@kgal-pan do we want to make this change on partner packs? IMO it makes sense to add. 
@talzich make sure that the partner packs dont have autoupdatedocker exclude flag if we do in fact bump their dockers

Reply:
@JudahSchwartz @talzich - Sure, it makes sense to make the change.